rskj-core/src/main/java/co/rsk/net/sync/PeersInformation.java

```diff
@@ -17,10 +15,12 @@
  */
 public class PeersInformation {
```

Comment:
There are at least three places in this class where you could reuse a method:

```java
private boolean peerNotExpired(SyncPeerStatus peer) {
    return !peer.isExpired(syncConfiguration.getExpirationTimePeerStatus());
}
```

Reply:
Code added :+1: